c_src/quicer_config.c

```diff
@@ -333,6 +371,24 @@ load_alpn(ErlNifEnv *env,
   return true;
 }
 
+bool
+load_verify(ErlNifEnv *env,
+            const ERL_NIF_TERM *options,
+            bool default_verify)
+{
+  ERL_NIF_TERM verify_atom;
+  int verify_atom_length;
```

Comment:
remove, not used.

Reply:
It actually is used. Do you think it should be inlined? The function is used both from quicer_config.c and quicer_listen.c.